.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,37 @@
+# Publish a GitHub Release to PyPI automatically
+# customized from:
+# https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+name: Publish Releases to PyPI
+
+on:
+  release:
+    type: [released]
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python distributions to PyPI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
```

Comment:
Should we do this for Python 3.7, 3.8 and 3.9 right away?

Reply:
I think the packages we create are independent of the python version so we only need to run this workflow once with one python version.